test/compilable/test18348.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -eu pipefail
+
+dir=${RESULTS_DIR}${SEP}runnable
+output_file=${dir}/test13774.sh.out
+
+faketty () { script -qfc "$(printf "%q " "$@")" /dev/null ; }
```

Comment:
Required because `-color=on` because DMD requires a TERM to exist for colorized output:

https://github.com/dlang/dmd/blob/419f13596a8fd44fd5a5301cb8db454d61be2b29/src/dmd/console.d#L172-L176